spec/plans/provision.fmf

```diff
@@ -171,7 +176,19 @@ example: |
 
         # Enable regular expression search
         cpu:
-            model_name: =~ .*AMD.*
+            model-name: =~ .*AMD.*
+
+        # Network interface
+        network:
+          - type: eth
+            vendor-name: =~ Broadcom.*
+            device-name: =~ NetXtreme II BCM.*
```

Comment:
the `=~` means a _regexp_ match I guess. The _space_ after `~` should be ignored? What if I want the space there? Shouldn't the space be removed?

Reply:
maybe we should not use `=~` at all - https://github.com/psss/tmt/issues/909

Reply:
Yeah, let's address the regexp issue separately in #909 as this syntax was not introduced here.